website/content/release-notes/_index.md

```diff
@@ -69,12 +115,7 @@ Bug Fixes:
   included tools were not usable.
   ([Issue #618](https://github.com/metallb/metallb/issues/618))
 
-This release includes contributions from bootjp / Yoshiaki Ueda, Brandon B.
-Jozsa, Carlos Goncalves, Christoph Raab, claudex, David Anderson, Dax McDonald,
-Etienne Champetier, Federico Paolinelli, Graeme Lawes, jlclx, Johannes
-Liebermann, Mark Gray, Maxim Makarov, Mike McKiernan, Mohamed Mahmoud, Ori
-Braunshtein, Rodrigo Campos, Russell Bryant, Sabina Aledort, t0b3, Till Adam,
-Tomofumi Hayashi, Trey Dockendorf, Utku Ozdemir, and Yves Mettier. Thank you!
+This release includes contributions from alphabet5, Andrea Panattoni, Brian_P, Carlos Goncalves, Carlos Gonçalves, Federico Paolinelli, Graeme Lawes, HeroCC, Ian Roberts, Lior Noy, Marco Geri, Mark Gray, Mohamed S. Mahmoud, Ori Braunshtein, Pumba98, rata, Russell Bryant, Sabina Aledort, Shivamani Patil and Tyler Auerbeck. Thank you!
```

Comment:
Woah! Such a privilege being highlighted twice, although not needed :smile: 
I can go by "Carlos Goncalves", avoids potential special character encoding issues

```suggestion
This release includes contributions from alphabet5, Andrea Panattoni, Brian_P, Carlos Goncalves, Federico Paolinelli, Graeme Lawes, HeroCC, Ian Roberts, Lior Noy, Marco Geri, Mark Gray, Mohamed S. Mahmoud, Ori Braunshtein, Pumba98, rata, Russell Bryant, Sabina Aledort, Shivamani Patil and Tyler Auerbeck. Thank you!
```

Reply:
Lol that's taken from the commit list.